.travis.yml

```diff
@@ -4,6 +4,14 @@ sudo: false
 
 matrix:
   include:
+    - php: 5.3
+      env: WITH_LOWEST=true
+    - php: 5.3
+      env: WITH_HIGHEST=true
+    - php: 5.4
+      env: WITH_LOWEST=true
+    - php: 5.4
```

Comment:
For reference, see [`composer.json`](https://github.com/codeclimate/php-test-reporter/blob/master/composer.json#L22), where we set the minimum platform requirement for PHP versions to be `>=5.3`.